types.d.ts

```diff
@@ -0,0 +1,78 @@
+declare module "gcs-browser-upload" {
+  type ChunkUploadData = {
+    totalBytes: number
+    uploadedBytes: number
+    chunkIndex: number
+    chunkLength: number
+  }
+
+  type UploadOptions = {
+    /** A unique ID for the upload */
+    id: string
+
+    /** GCS resumable URL */
+    url: string
+
+    /** The File to upload */
+    file: File
+
+    /**
+     * Chunk size must be a multiple of 262144
+     * @default 262144
+     */
+    chunkSize?: number
+
+    /**
+     * Storage mechanism used to persist chunk metadata
+     * @default window.localStorage
+     */
+    storage?: Storage
+
+    /**
+     * Content type of the file being uploaded
+     * @default 'text/plain'
+     */
+    contentType?: string
+
+    /** A function that will be called with progress information */
+    onChunkUpload?: (data: ChunkUploadData) => void
+  }
+  export default class Upload {
+    static errors: typeof UploadErrors
+
+    constructor(config: UploadOptions, allowSmallChunks?: boolean)
+
+    start(): Promise<void>
+    pause(): void
+    unpause(): void
+    cancel(): void
+  }
+
+  namespace UploadErrors {
+    declare class DifferentChunkError extends Error {
```

Comment:
![image](https://user-images.githubusercontent.com/3920879/226977885-db9c3a29-1b4c-4565-a673-d809b96c9967.png)
i get this when i `npm i`, i think you dont need to `declare` again